python/paddle/v2/fluid/tests/test_maxout_op.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+def maxout_forward_naive(input, groups,num_channels):
```

Comment:
The `num_channels ` is not used, please to remove it.

Reply:
done